hack/terraform-gcr-init/README.md

```diff
@@ -0,0 +1,26 @@
+terraform-gcr-init
+==================
+
+This can be used as a Terraform module to set up a Google Project
+for each developer working on cluster-api-provider-aws as a place for them
+to push container images.
```

Comment:
Should we add a warning that images are made public by default?

Reply:
Almost certainly. Thx.